gui/resources.xml

```diff
@@ -1779,39 +1779,33 @@ Vocabulary
       </child>
       <child>
         <object class="GtkRadioAction" id="ColorPickModeH">
-          <property name="label" translatable="yes" context="Menu→Edit→Current Mode (labels)">Pick Color Hue</property>
-          <property name="short-label" translatable="yes" context="Toolbar (short labels)">Pick ColH.</property>
-          <property name="tooltip" translatable="yes" context="Toolbar (tooltips)">Pick Color: set the painting color Hue from screen pixels.</property>
           <property name="icon-name">mypaint-pick-color-h-symbolic</property>
           <property name="group">SymmetryEditMode</property>
           <property name="value">11</property>
+          <!-- NO TOOLTIP OR LABEL. -->
```

Comment:
Should every action have a menu item?  Seems like 4 colorpick modes in the menu is overkill but not sure